package.json

```diff
@@ -271,5 +271,8 @@
     "tools/@*/*",
     "apps/*"
   ],
+  "config": {
+    "o3rMetrics": true
```

Comment:
are we ok to set that by default in the repo ?
anyone cloning and building the project will unwillingly send data when executing builders